Description: A patch for Makefile.am before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 It was made to handle undefined non-weak symbols. Now is deprecated.
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07
Bug-Debian: http://bugs.debian.org/195970

--- autotrace-0.31.1.orig/Makefile.am
+++ autotrace-0.31.1/Makefile.am
@@ -127,7 +127,9 @@ autotrace_SOURCES=\
 
 INCLUDES = $(MAGICK_CFLAGS) $(LIBPSTOEDIT_CFLAGS)
 autotrace_LDADD =				\
-		libautotrace.la			\
+		libautotrace.la
+
+libautotrace_la_LIBADD = \
 		$(MAGICK_LDFLAGS)		\
 		$(LIBPNG_LDFLAGS)		\
 		$(LIBSWF_LDFLAGS)		\
